Makefile

```diff
@@ -145,11 +147,13 @@ endif
 
 .PHONY: test
 test:
+	$(MAKE) -C ./tests/lenses build
 	gotestsum --format pkgname -- ./... $(TEST_FLAGS)
 
 # Only build the tests (don't execute them).
 .PHONY: test\:build
 test\:build:
+	$(MAKE) -C ./tests/lenses build
```

Comment:
todo: please remove this step isn't for running the tests. It's only to build them. If would want to do this then the `lenses` `Makefile` should include a rule similar to `-run=nope`. Even then according to our discussion on discord I would make a new rule to include that under `make test:build-all`